x-pack/plugins/file_upload/public/util/indexing_service.js

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { http } from './http_service';
+import chrome from 'ui/chrome';
+import { i18n } from '@kbn/i18n';
+import { indexPatternService } from '../../../maps/public/kibana_services';
```

Comment:
This is going to cause problems when the plugin is migrated to new platform. You will not be able to deep link into other plugins like this. Lets initialize a indexPatternService from within this plugin when it starts

Reply:
Yep! That's actually an import auto-complete error. Nice catch! There's already a local service, I've updated it to import this instead